eos-link-host-mali-driver

```diff
@@ -22,14 +22,45 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 EXTENSION_REF=org.freedesktop.Platform.GL.host/arm/1.6
-MALI_DIR=/usr/lib/mali400/arm-linux-gnueabihf
 
-DEST_DIR="/var/lib/flatpak/extension/$EXTENSION_REF"
-mkdir -p $(dirname "$DEST_DIR")
-ln -sf "$MALI_DIR" "$DEST_DIR"
+#
+# create_flatpak_gl_host_extension MALI_DIR FLATPAK_DIR
+#
+# Creates a flatpak GL host extension in $FLATPAK_DIR from $MALI_DIR directory
+# MALI_DIR needs to contain libMali.so and libMaliWrapper.so libraries.
+#
+function create_flatpak_gl_host_extension {
+    MALI_DIR=$1
```

Comment:
What you are doing here is overwrite (with the same value, BTW) the variable declared on line 59. Despite not leading to problems here, is better to avoid leaving a trap for future changers of this code. Either use `local` here or choose a different name.